extension/src/LanguageFunctions.ts

```diff
@@ -139,14 +139,14 @@ export async function findNextUnTranslatedText(searchCurrentDocument: boolean):
         if (useExternalTranslationTool) {
             searchFor = targetStateActionNeededKeywordList();
         } else {
-
             searchFor = [TranslationToken.Review, TranslationToken.NotTranslated, TranslationToken.Suggestion];
         }
         let searchResult = await findClosestMatch(xlfUri, startOffset, searchFor);
         if (searchResult.foundNode) {
             await DocumentFunctions.openTextFileWithSelection(xlfUri, searchResult.foundAtPosition, searchResult.foundWord.length);
             return true;
         }
+        removeCustomNotesFromFile(xlfUri);
```

Comment:
I think that this should only be done if `searchCurrentDocument` is false, don't you think?

Reply:
naah, I just saw that the condition in removeCustomNoteFromFile takes care of this. Splendid!